src/modules/createGameScores.js

```diff
@@ -0,0 +1,34 @@
+import clearInputs from './clearInputs.js';
+import getGameID from './getGameID.js';
+
+const id = getGameID();
+const url = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${id}/scores/`;
+
+const createGameScore = async () => {
+  const user = document.getElementById('user-input').value;
+  const score = document.getElementById('score-input').value;
+
+  const postData = { user, score };
+
+  const requestOptions = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(postData),
+  };
+
+  try {
+    const response = await fetch(url, requestOptions);
+
+    if (!response.ok) {
+      throw new Error('Network response was not ok');
+    }
+
+    clearInputs();
+  } catch (error) {
+    // return error;
+  }
+};
```

Comment:
- [ ] Please consider removing the commented code from our codebase to maintain a clean and professional code repository.

- [OPTIONAL] I wanted to suggest a slight improvement in our code. Instead of leaving the `catch` block empty, consider throwing an error within it. This way, you can proactively handle the error at a later point in the code.